client/src/components/cards/CardsIndex.jsx

```diff
@@ -1,9 +1,23 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import Form from './Form';
-import * as cardsActions from '../../actions/cards';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
we should probably setup an scss directory and start including our stylesheets and libraries in there, import them on the index page

Reply:
Ok, I'll create another issue for SCSS. For now I'm adding some basic styling to our material components, following the examples in the material design documentation and examples. 
[Card component demo](https://material-ui.com/demos/cards/).